src/Behat/Behat/Output/Statistics/Statistics.php

```diff
@@ -65,6 +65,14 @@
      */
     public function __construct()
     {
+        $this->resetAllCounters();
+
+        $this->timer = new Timer();
+        $this->memory = new Memory();
+    }
+
+    public function resetAllCounters()
```

Comment:
If base `Statistics` doesn't work for JUnit formatter I'd rather JUnit had it's own version of `Statistics` than everyone else to depend on interface they don't fully use.